src/actions.js

```diff
@@ -70,7 +70,7 @@ export function formatMedicalItemOrServiceGQL(mm, ms) {
     ${ms.maximumAmount ? `maximumAmount: "${ms.maximumAmount}"` : ""}
     ${ms.careType ? `careType: "${formatGQLString(ms.careType)}"` : ""}
     ${ms.frequency ? `frequency: "${ms.frequency}"` : ""}
-    ${ms.patientCategory ? `patientCategory: ${ms.patientCategory}` : ""}
+    ${`patientCategory: ${ms.patientCategory}`}
```

Comment:
I guess it is assumed to be required in canSave?

Reply:
I saw that canSave does not cover categories: 
![image](https://github.com/openimis/openimis-fe-medical_js/assets/109145288/318db64f-3dba-4b7a-9135-5113d94aee3c)
And BE mutation also does not require those categories, so I assumed that those checkboxes are optional. This could be confirmed by @delcroip or @dragos-dobre .

Reply:
I've got answer from Patrick: 
![image](https://github.com/openimis/openimis-fe-medical_js/assets/109145288/c4ce072c-6e75-4b74-8578-3852594b404b)
I'm moving ticket to draft and I'll rework this approach to be more user friendly.